src/ranking/aggregation.ts

```diff
@@ -22,7 +22,7 @@ export function score(specM: SpecQueryModel, schema: Schema, opt: QueryConfig):
 function aggregationQualityFeature (specM: SpecQueryModel, _: Schema, __: QueryConfig): FeatureScore {
   const encodings = specM.getEncodings();
   if (specM.isAggregate()) {
-    const isRawContinuous = (encQ: EncodingQuery) => {
+    const isRawContinuous = (encQ) => {
```

Comment:
I don't think we should cheat by removing `: EncodingQuery` here :p.  better property check it!